src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -99,7 +103,7 @@ def _create_cont(self):
 
         # create container
         # self.container.create()
-        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
+        env = self.mdtest_cmd.get_default_env()
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'manager_cmd' in method call
(pylint-no-value-for-parameter) No value for argument 'attach_info' in method call